linux_os/guide/services/ssh/ssh_server/sshd_use_strong_kex/rule.yml

```diff
@@ -0,0 +1,38 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Use Only Strong Key Exchange algorithms'
+
+description: |-
+    Limit the Key Exchange to strong algorithms.
+    The following line in <tt>/etc/ssh/sshd_config</tt> demonstrates use
+    of those:
+    <pre>KexAlgorithms ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,
+    diffie-hellman-group-exchange-sha256,diffie-hellman-group16-sha512,diffie-hellman-group18-sha512,diffie-hellman-group14-sha256</pre>
```

Comment:
You should use a variable here because you parametrized the rule by a variable.

Reply:
Thanks for the feedback :bow: - should be handled in 6f0acb37fc